### DISCUSSION TITLE
version changelog

### DISCUSSION THREAD

**Discussion by DarioGui** [YetiForceCompany/YetiForceCRM#16407] [How to]
Hi everyone! Sorry but I don't understand where to look.
I saw that 6.4 is in its third version; where can I find the changelog to see what are the corrections between the various releases of versions of 6.4?
sorry again

**Comment by mariuszkrzaczkowski** [comment]
The changelog for large versions is here:
https://yetiforce.com/en/knowledge-base/documentation/developer-documentation/category/changes-2

Small update packages, e.g. [6.3.0_ServicePack](https://github.com/YetiForceCompany/UpdatePackages/tree/master/YetiForce%20CRM%206.x.x/6.3.0_SecurityFix), [6.2.0_SecurityFix](https://github.com/YetiForceCompany/UpdatePackages/tree/master/YetiForce%20CRM%206.x.x/6.2.0_SecurityFix)

The list of changes made is always shown during the installation:
![image](https://user-images.githubusercontent.com/10198654/187156721-58a0677b-9744-4281-a62c-deac984df246.png)

This can also be seen in the `license.txt` file of a small upgrade package, e.g.
https://github.com/YetiForceCompany/UpdatePackages/blob/master/YetiForce%20CRM%206.x.x/6.4.0_ServicePack/license.txt

**Comment by DarioGui** [comment]
I had seen the major ones, I missed the minor ones or the fixes.
This way I found version 3
but for version 1 and version 2 where do I find them?
in the update packages I found only the latest.
thank you very much

**Reply by mariuszkrzaczkowski** [reply]
See file history
https://github.com/YetiForceCompany/UpdatePackages/commits/master/YetiForce%20CRM%206.x.x/6.4.0_ServicePack/license.txt
example:
6.4.0_ServicePack_v3 https://github.com/YetiForceCompany/UpdatePackages/commit/d5cfb8b92d03bb135104bd277d1eefac8f0a5a88

**Comment by DarioGui** [comment]
ah ... I hadn't thought of looking in the history :)
thank you
